lang/src/org/partiql/lang/ast/ExprNodeToStatement.kt

```diff
@@ -99,10 +99,11 @@ fun ExprNode.toAstExpr(): PartiqlAst.Expr {
                 }
             }
             is CallAgg -> {
-                val symbol1 = (node.funcExpr as? VariableReference)?.id
+                val symbol1 = (node.funcExpr as? VariableReference)
```

Comment:
When working on a change to [`TestBase.kt`](https://github.com/partiql/partiql-lang-kotlin/pull/341/commits/7c175faecb625386367058ac2a398f278ce2812d#diff-32f794c776727f8be85a860d0f8157e373f1ba9951d2b261d9898e396701d359), some of the tests were failing because of missing metas. Changes to this file adds `metas` that were previously not passed.